mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -167,6 +173,10 @@ CLASS net/minecraft/class_1752 net/minecraft/block/Blocks
 	FIELD field_7424 PLANKS Lnet/minecraft/class_197;
 	FIELD field_7425 SAPLING Lnet/minecraft/class_197;
 	FIELD field_7426 BEDROCK Lnet/minecraft/class_197;
+	FIELD field_7427 FLOW_WATER Lnet/minecraft/class_153;
+	FIELD field_7428 WATER_BLOCK Lnet/minecraft/class_197;
```

Comment:
WATER

Reply:
I can't, it's already taken

Reply:
h o w

Reply:
can you check if all blocks have the right name?